routes/v1.js

```diff
@@ -149,37 +228,483 @@ api.declare({
   output:         SCHEMA_PREFIX_CONST + 'get-worker-type-response.json#',
   title:          "Get Worker Type",
   description: [
-    // Document what this method does
+    "Retreive a WorkerType definition"
   ].join('\n')
 }, function(req, res) {
   var ctx         = this;
   var workerType  = req.params.workerType;
 
+  if(!req.satisfies({
+    workerType:       workerType
+  })) {
+    return; // by default req.satisfies() sends a response on failure, so we're done
+  }
+
+  var p = ctx.WorkerType.load({workerType: workerType});
+  
+  p = p.then(function(worker) {
+    return res.reply(worker.json());
+  });
+
+  p = p.catch(function(err) {
+    errorHandler(err, res, workerType);
+    return err;
+  });
+
+  return p;
+
 });
 
-// List workerTypes
+
+// Delete workerType
+// TODO: send a pulse message that a worker type was removed
+api.declare({
+  method:         'delete',
+  route:          '/worker-type/:workerType',
+  name:           'removeWorkerType',
+  deferAuth:      true,
+  // TODO: Should we have a special scope for workertype removal?
+  scopes:         ['aws-provisioner:get-worker-type:<workerType>'],
```

Comment:
It's weird that we'd have a different set of scopes for creating, updating, viewing and removing.

Shouldn't we only have two: view and mutate?  Why would we ever let someone who can create a workertype not update it, or let someone remove one which they cannot create....